Makefile

```diff
@@ -577,40 +577,82 @@ distclean: clean
 html: $(HTML)
 
 # Rules to attach JavaScript and CSS assets to the head
-search/index.html: search/index.html.in .make-asset-block .make-template-block
+search/index.html: .make-asset-block .make-template-block
 	sed -e '/%ASSETS%/ {' -e 'r .make-asset-block' -e 'd' -e '}' \
 		-e '/%TEMPLATES%/ {' -e 'r .make-template-block' -e 'd' -e '}' \
-		search/index.html.in > search/index.html
+		search/search.html.in > search/index.html
 
-login/index.html: login/index.html.in .make-asset-block
+login/index.html: .make-asset-block
```

Comment:
and `index.html.in` should be a dependency here and throughout the rest of the rules, right?